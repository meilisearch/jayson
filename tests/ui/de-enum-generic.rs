use jayson::Deserialize;

#[derive(Deserialize)]
enum Enum<const T: i32> {
    Variant,
}

fn main() {}
