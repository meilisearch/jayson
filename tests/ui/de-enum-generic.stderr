error: Enums with generics are not supported
 --> tests/ui/de-enum-generic.rs:3:10
  |
3 | #[derive(DeserializeFromValue)]
  |          ^^^^^^
  |
  = note: this error originates in the derive macro `Jayson` (in Nightly builds, run with -Z macro-backtrace for more info)
